mpqp/tools/generics.py

```diff
@@ -205,4 +223,49 @@ class SimpleClassReprABCMeta(SimpleClassReprMeta, ABCMeta):
 
 
 class SimpleClassReprABC(metaclass=SimpleClassReprABCMeta):
-    pass
\ No newline at end of file
+    pass
```

Comment:
Should we put a Doc for this class, cauz I don't understand directly what it is useful for from the name

Reply:
done